app/src/main/java/org/mozilla/focus/utils/FeatureFlags.kt

```diff
@@ -3,9 +3,11 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 package org.mozilla.focus.utils
 
+import org.mozilla.focus.BuildConfig
+
 /**
  * Feature flag for MVP
  */
 object FeatureFlags {
-    const val isMvp = false
+    var isMvp = BuildConfig.BUILD_TYPE.contains("debug")
```

Comment:
nit: Wouldn't `val isMvp = BuildConfig.DEBUG` be enough?
Same thing, in a bit shorter format.